src/appleseed.cli/progresstilecallback.cpp

```diff
@@ -89,13 +102,33 @@ namespace
             const size_t total_pixels = frame->image().properties().m_pixel_count;
 
             // Print a progress message.
-            LOG_INFO(m_logger, "rendering, %s done", pretty_percent(m_rendered_pixels, total_pixels).c_str());
+            //LOG_INFO(m_logger, "rendering, %s done", pretty_percent(m_rendered_pixels, total_pixels).c_str());
```

Comment:
Don't forget to remove dead code.

Reply:
I keep missing things for some reason! This time it is perfect.